scripts/irods/configuration.py

```diff
@@ -238,16 +238,17 @@ def validate_configuration(self):
                     'path': paths.server_config_path()},
                 'VERSION': {
                     'dict': self.version,
-                    'path': paths.version_path()},
-                'hosts_config': {
-                    'dict': self.hosts_config,
-                    'path': paths.hosts_config_path()},
-                'host_access_control_config': {
-                    'dict': self.host_access_control_config,
-                    'path': paths.host_access_control_config_path()},
-                'service_account_environment': {
-                    'dict': self.client_environment,
-                    'path': self.client_environment_path}}
```

Comment:
@trel I think this is part of the schema verification step. These can be removed now that they are part of server_config.json, right?

Reply:
correct, we will not be supporting the 'old' style/files - we just migrate, and then validate the new things only.

oh... but not service_account_environment...?  hmmm

Reply:
I don't know what that is.

Reply:
it defines what a service account environment file looks like...

`/var/lib/irods/configuration_schemas/v3/service_account_environment.json`
which describes
`/var/lib/irods/.irods/irods_environment.json`

we need that one to remain.

Reply:
I've restored the service_account_environment part.